snippets/server-platforms/php/dependencies.md

```diff
@@ -1,5 +1,5 @@
 To install dependencies, run the following
 
 ```bash
-composer require auth0/auth0-php:"~5.0"
+composer require auth0/auth0-php:"~7.0"
```

Comment:
Should we also update the requirements for the quickstart considering the [PHP support ](https://www.php.net/supported-versions.php)?

![image](https://user-images.githubusercontent.com/2146903/95593138-97c39600-0a49-11eb-8a8a-cc66d7812971.png)

Reply:
Good catch, thanks! I'll update to fix that as well. That PHP version needs bumped to 7.1 as well as that's the minimum supported by our PHP SDK. I'm on it.